data/JimmyTron.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "Jimmy Tron",
+  "bio": "**Python Web Developer** | **Junior DevOps Engineer** | **Tech support Specialist** | ** Coding Instructor** | **Tech Community Champion** ",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/jimmytron_ke",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/james-oduor/",
+      "icon": "FaLinkedIn"
```

```suggestion
      "icon": "FaLinkedin"
```
Can you please make the above changes as the icon is case sensitive and in the documentation, it's `FaLinkedin `and not `FaLinkedIn`